docs/configuration/istio/ingress.md

```diff
@@ -0,0 +1,83 @@
+3. **(Optional)** Passthrough Gateway - This gateway allows mesh ingress without TLS termination performed by Istio. This could be useful for applications that need to (or currently) handle their own TLS termination. This gateway used to be a default component of UDS Core but is no longer deployed by default. To deploy this gateway, you must specify `istio-passthrough-gateway` as an `optionalComponent` in your UDS Bundle configuration.
+
+### Enable Passthrough Gateway
+
+In order to enable the Passthrough Gateway, you must specify `istio-passthrough-gateway` as an `optionalComponent` in your UDS Bundle configuration. Here is an example of how to do this:
+
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-passthrough
+  description: A UDS example bundle for packaging UDS core with the passthrough gateway enabled
+  version: "0.0.1"
+
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.23.0-upstream
+    # You must specify the istio-passthrough-gateway as an optionalComponent or else it will not be deployed
+    optionalComponents:
+      - istio-passthrough-gateway
+```
+
+### Configure Domain Name and TLS for Istio Gateways
+
+By default, the UDS Core Istio Gateways are set up to use the `uds.dev` domain and have a valid TLS certificate packaged.  You will want to change the domain name for your environment and provide a valid TLS certificate for this domain.
+
+You can set the TLS certs via overrides in a [UDS Bundle](https://uds.defenseunicorns.com/bundles/) (see below).
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-cert-override
+  description: A UDS example bundle for packaging UDS core with a custom TLS certificate
+  version: "0.0.1"
+  
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.23.0-upstream
+    overrides:
+      istio-admin-gateway:
+        uds-istio-config:
+          variables:
+            - name: ADMIN_TLS_CERT
+              description: "The TLS cert for the admin gateway (must be base64 encoded)"
+              path: tls.cert
+            - name: ADMIN_TLS_KEY
+              description: "The TLS key for the admin gateway (must be base64 encoded)"
+              path: tls.key
+      istio-tenant-gateway:
+        uds-istio-config:
+          variables:
+            - name: TENANT_TLS_CERT
+              description: "The TLS cert for the tenant gateway (must be base64 encoded)"
+              path: tls.cert
+            - name: TENANT_TLS_KEY
+              description: "The TLS key for the tenant gateway (must be base64 encoded)"
+              path: tls.key
+```
+The environment variables `ADMIN_TLS_CERT`, `ADMIN_TLS_KEY`, `TENANT_TLS_CERT`, and `TENANT_TLS_KEY` should be base64 encoded strings of the TLS certificate and key for the admin and tenant gateways respectively.
```

Comment:
Formatting, should add a newline here (don't recall if the docs site will enforce this during lint, but probably good to add regardless.